src/test/datascience/notebook/nonPythonKernels.vscode.test.ts

```diff
@@ -101,6 +111,10 @@ suite('DataScience - VSCode Notebook - Kernels (non-python-kernel) (slow)', () =
         await openNotebook(api.serviceContainer, testJuliaNb.fsPath);
         await waitForKernelToGetAutoSelected('julia');
     });
+    test('Automatically pick csharp kernel when opening a Julia Notebook', async () => {
+        await openNotebook(api.serviceContainer, testJuliaNb.fsPath);
```

Comment:
I'm confused, why would we want to automatically pick a C# kernel when opening a Julia notebook?

Reply:
Fixed